assets/scripts/dialogs/__tests__/GeotagDialog.test.js

```diff
@@ -2,9 +2,53 @@
 import React from 'react'
 import GeotagDialog from '../GeotagDialog'
 import { shallow } from 'enzyme'
+import { getRemixOnFirstEdit } from '../../streets/remix'
 
 // Mock dependencies that could break tests
-jest.mock('../../streets/remix', () => {})
+jest.mock('../../streets/remix', () => ({
+  getRemixOnFirstEdit: jest.fn()
+}))
+
+function getTestComponent (addressInformation = null, street = null) {
+  const testMarker = { lat: 0, lng: 0 }
+  let testAddressInfo = {
+    street: 'foo',
+    id: 'foo'
+  }
+  let testStreet = {
```

Comment:
I think it's easier to say something like this (and you still get to use `const`):

```js
const testStreet = street || { ... }
```